cdhweb/pages/wagtail_hooks.py

```diff
@@ -9,12 +9,23 @@ def global_admin_css():
     """Add wagtail custom admin CSS."""
     return format_html(
         '<link rel="stylesheet" href="{}">',
-        static("wagtailadmin/css/custom.css")
+        static("wagtailadmin/css/custom.css"),
     )
 
 
 @hooks.register("register_log_actions")
 def register_exodus_log_action(actions):
     """Add a custom PageLogEntry action to mark page exodus from Mezzanine."""
-    actions.register_action("cdhweb.exodus", "Exodus",
-                            "Migrated from cdhweb v2")
+    actions.register_action("cdhweb.exodus", "Exodus", "Migrated from cdhweb v2")
+
+
+@hooks.register("before_edit_page")
+def create_redirect_on_slug_change(request, page):
```

Comment:
Please add a comment here linking to the wagtail recipe you used and note whether it was copied exactly or adapted in any way.